package.json

```diff
@@ -15,15 +15,17 @@
   },
   "scripts": {
     "build:storybook": "build-storybook",
+    "build:design-tokens": "npm run clean:design-tokens && node ./packages/design-tokens/scripts/build.js",
     "clean": "rimraf dist",
+    "clean:design-tokens": "rm -rf ./dist/packages/design-tokens",
     "commitlint": "commitlint -e",
     "create:component": "./scripts/create-component",
     "deploy:storybook": "storybook-to-ghpages",
     "predist": "npm run clean",
-    "dist": "tsc -p tsconfig.dist.json",
+    "dist": "tsc -p tsconfig.dist.json && npm run build:design-tokens",
     "lint": "tslint -c tslint.json -p tsconfig.json",
     "setup:git-hooks": "./scripts/pre-install",
-    "start": "NODE_ENV=development npm run storybook --root ./ ",
+    "start": "NODE_ENV=development npm run build:design-tokens && npm run storybook --root ./ ",
```

Comment:
Yes, this is weird, but we can just import design tokens as a package once we use Lerna to manage multiple packages in this repo.
Then we can remove all the design token changes in here